docs/platform/release-notes/tizen-6-5-m2.md

```diff
@@ -0,0 +1,400 @@
+    - WebAssembly MVP features have been added.
+    - JS CodeCache has been optimized.
+- JS based lightweight backend service FW
+  - Lightweight node.js has been added.
+
+#### Fixes
+
+- Lightweight Web  Engine
+  - (Grid/Flex) Layout defects have been fixed.
+  - File defects have been fixed.
+
+
+### Tizen .NET
+
+#### New and changed features
+
+- TizenFx
+  - Tizen.NUI
+    - LinearLayout.LinearAlignment has been deprecated.
+    - Navigator Popped event has been added.
+    - Page Appear/Disappear event has been added.
+    - Argument of TimePeriod has been changed.
+    - LinearLayout.Horizontal/VerticalAlignment has been added.
+    - AdjustViewSize has been added to ImageView.
+    - Page Transition has been added.
+    - The name of ImageView Property has been changed.
+    - Text Selection has been added to TextField, TextEditor.
+    - Input Filter has been added to TextField, TextEditor.
+    - Copy, Cut, and Paste have been added to TextField, TextEditor.
+    - BorderlineWidth/Color/Offset has been added to NUI.BaseComponents.View.
+    - RiveAnimationView has been added.
+    - Support for IME Window has been added.
+    - WebView has been added.
+    - CameraView has been added.
+    - Picker has been added.
+    - ResizePolicy has been deprecated.
+  - Tizen.Multimedia
+    - New Inference configuration has been added.
+    - Legacy Inference configuration has been deprecated.
+    - Support for Video & UI Sync has been added.
+    - Support for HEIF image has been added.
+    - ConnectAsync has been added.
+    - Support for EBook Format has been added.
+    - Additional GATT API set has been added.
+    - WebRTC has been added.
+    - Support for CameraPixelFormat (VP8, VP9) has been added.
+    - Additional VideoMimeTypes have been added.
+  - Tizen.Applications
+    - Cion has been added.
+    - New constructor and method has been added to RPCPort.
+    - Parcel.Header has been added to RPCPort.
+    - Support for Resource Control has been added.
+  - Tizen.Uix.Tts
+    - Errors and API set for Screen Reader Option have been added.
+  - Tizen.Uix.InputMethod
+    - IME Resizing has been added.
+  - Tizen.MachineLearning
+    - Support for Neural Network Framework and HW Type has been added.
+  - Tizen.Security
+    - OCSP has been deprecated.
+
+
+### Toolchain
+
+#### New and changed features
+
+- Hardware-assisted AddressSanitizer (HWASan)
+  - GCC/ Libsanitizer : HWASan feature has been backported and tuned for Tizen.
+  - HWASan compiler options have been applied.
+  - HWASan runtime options have been applied to build and runtime environment.
+  - Support for HWASan has been added to glibc and packages.
+- Large File Support (LFS)
+  - LFS support has been added to aarch64.
+- New architecture(Armv7hl) support
+  - Build support has been added for 32-bit hard float ABI.
+
+#### Fixes
+
+- Binutils/ LTO bug #25355
+  - LTO plugin bugfix has been backported into binutils, ld, and bfd.

```

Comment:
Some places they are in caps some in title case, recommend to keep it consistent:
binutils, ld, and bfd.

Reply:
@annie-samsung Actually sub-modules {binutils, ld, and bfd} are included in `Binutils`  module. So, I want keep it as it is.
Thank you!